staging/cse/windows/azurecnifunc.tests.suites/AzureCNI.Default.OverlayDualStack.conflist

```diff
@@ -0,0 +1,47 @@
+{
+  "cniVersion": "0.3.0",
+  "name": "azure",
+  "adapterName": "",
+  "plugins": [
+    {
+      "type": "azure-vnet",
+      "mode": "bridge",
+      "bridge": "azure0",
+      "executionMode": "v4swift",
+      "capabilities": {
+        "portMappings": true,
+        "dns": true
+      },
+      "ipam": {
+        "type": "azure-cns",
+        "mode": "dualStackOverlay"
+      },
+      "dns": {
+        "Nameservers": [
+          "10.0.0.10",
```

Comment:
Q: Do we need to update dns servers for ipv6?

Reply:
I don't think right now because we don't support accepting a kube dns service IP from users in IPv6 format. Even in dual-stack clusters we only accept the v4 one.